sass/components/_toggle.scss

```diff
@@ -30,13 +30,13 @@ $toggle-default-font-size: .75rem;
 @mixin toggle-mixin($size: $toggle-default-size, $margin: $toggle-default-margin, $font-size: $toggle-default-font-size){
     // color: $color;
     // background: $bg;
-    margin: 0 $margin;
     padding: 0;
     position: relative;
     border: none;
     height: $size;
-    width: $size * 2;
+    min-width: $size * 2;
     border-radius: $size;
+    float: right;
```

Comment:
Note that this will globally change the alignment of these toggles. @michaelgamble can you confirm that this is your intent?